go/client/cmd_login.go

```diff
@@ -24,6 +25,14 @@ func NewCmdLogin(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command
 			Name:  "s, switch",
 			Usage: "switch out the current user for another",
 		},
+		cli.StringFlag{
+			Name:  "paperkey",
+			Usage: "DANGEROUS: automatically provision using this paper key",
+		},
+		cli.StringFlag{
+			Name:  "devicename",
+			Usage: "Device name used in automated provisioning",
```

Comment:
update the online docs to describe what these two params are, how they work together, etc.

Reply:
Where should I put that stuff? There's no existing document afaik

Reply:
Whatever is output in `keybase login --help` should be expanded upon.